internal/host/statemachine.go

```diff
@@ -384,7 +384,7 @@ func NewHostStateMachine(th *transitionHandler) stateswitch.StateMachine {
 
 	var isSufficientForInstall = stateswitch.And(If(HasMemoryForRole), If(HasCPUCoresForRole), If(BelongsToMachineCidr),
 		If(IsHostnameUnique), If(IsHostnameValid), If(IsAPIVipConnected), If(BelongsToMajorityGroup),
-		If(AreOcsRequirementsSatisfied), If(AreLsoRequirementsSatisfied), If(AreCnvRequirementsSatisfied))
+		If(AreOcsRequirementsSatisfied), If(AreLsoRequirementsSatisfied), If(AreCnvRequirementsSatisfied), If(HasNetworkLatencyRequirementForRole), If(HasPacketLossRequirementForRole))
```

Comment:
Maybe SufficientNetworkLatencyRequirementForRole is better?

Reply:
Will update. Thanks!